index.js

```diff
@@ -400,6 +400,24 @@
   //. ```
   S.K = def('K', [a, b], function(x, y) { return x; });
 
+  //. ### Function
+
+  //# flip :: (a -> b -> c) -> b -> a -> c
+  //.
+  //. Takes a binary function and two values of any type and returns
```

Comment:
I just realized that _of any type_ could be misread as _of any **one** type_. The type signature clarifies things, of course, but this suggests we could drop _of any type_ from the sentence without changing its meaning. Let's do so.